src/routes/spa/spa-router.ts

```diff
@@ -12,11 +13,14 @@ SpaRouter.use("/static", Static(path.join(rootPath, 'spa/build/static')));
 
 //Because it is Single Page App, for all routes to /spa/screen1 , /spa/screen1/step2 should render the spa index.html anyway
 let indexHtmlContent: string = "";
-SpaRouter.use("/*", async function SpaIndexHtml(_, res) {
+SpaRouter.use("/*", async (req, res) => {
 	if (!indexHtmlContent) {
+		const jiraHost = req.query.xdm_e?.toString();
```

Comment:
I thought `xdm_e` was being deprecated
https://community.developer.atlassian.com/t/deprecation-of-xdm-e-usage/32768

But thats 3 years old so maybe they stopped caring?